Assets/Scripts/Possession.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Possession : MonoBehaviour
+{
+
+    private Player player;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        player = gameObject.GetComponent<Player>();   
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    void OnTriggerStay(Collider other)
+    {
+        if(other.gameObject.tag == "Dead" && Input.GetKeyDown("space")){

```

Comment:
Be sure to use the official controls for possession per Edit > Project Settings > Input. This will allow it to work with both keyboard and controller.

Reply:
Fixed thanks